### PR TITLE
Slow tests under a feature flag 

### LINKED ISSUE #1877: Slow tests under a feature flag
**Issue summary**

It would be great to put slow tests under a dedicated feature flag, so a regular `cargo test` is more convenient to use during development. Currently, e.g. this test takes significant amount of time:
```
test test_download_location_of_proof_parameter_files_env has been running for over 60 seconds
```
Naturally, the CI should run both slow and fast tests. We could say that 1s is the cut-off line (any test running for longer is slow). While 1s is not a particularly long period, 100 tests of 1s [would risk injuries](https://xkcd.com/303/) during Forest development.

Perhaps there is a dedicated test framework for that?

**Other information and links**

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes #1877

**Other information and links**

### CODE REVIEW

File: forest/cli/tests/config_tests.rs [hunk 1]
```diff
@@ -3,7 +3,9 @@
 use assert_cmd::Command;
 use forest_cli_shared::cli::{Client, Config};
 use rand::Rng;
+#[allow(unused_imports)]
 use std::{fs::read_dir, io::Write, net::SocketAddr, path::PathBuf, str::FromStr};
+#[allow(unused_imports)]
```
Comment (creativcoder) [inline forest/cli/tests/config_tests.rs:8]: can get rid of this, if we do `tempdir::TempDir` within `#[cfg(feature = "slow_tests")]`, likewise where applicable.

Comment (tyshko5) [reply forest/cli/tests/config_tests.rs]: fixed

**Review by lemmih — APPROVED** [review]
This will run parts of the test suite twice during CI, right? How much will that slow us down?

**Comment by tyshko5** [comment]
> This will run parts of the test suite twice during CI, right? How much will that slow us down?

Yes, something south of 5 seconds (pessimistically)

**Review by hanabi1224 — APPROVED** [review]

**Review by hanabi1224 — APPROVED** [review]

**Pull request merged**